Tests/SourceKitTests/SwiftCompletionTests.swift

```diff
@@ -142,8 +142,7 @@ final class SwiftCompletionTests: XCTestCase {
       XCTAssertEqual(test.filterText, "test(a:)")
       // FIXME:
       XCTAssertNil(test.textEdit)
-      // FIXME: should be "a" in the placeholder.
-      XCTAssertEqual(test.insertText, "test(a: ${1:value})")
+      XCTAssertEqual(test.insertText, "test(a: ${1:Int})")
```

Comment:
We should add a test for completion where the type and display name are not the same:
```
func test(_ b: Int) // <#T##b: Int##Int#>
```